sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -90,6 +90,19 @@ private[sql] case class JDBCRelation(
 
   override val schema: StructType = JDBCRDD.resolveTable(url, table, properties)
 
+  // Check if JDBCRDD#compileFilter can accept input filters
```

Comment:
This is a little nit I was pointed out before :). We might have to write a command with [[ ]] for Scala class and with \` \` or with nothing for functions.